src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Emblema+One&family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images\logo.png"
+          alt="moyo"
+          class="img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
```

Comment:
I suggest you putting 1 blank line between sibling tags (when they have some content) to improve readability. But not parent and its child
```suggestion
      </a>

      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>

          <li class="nav__item">
```